frontend/tests/e2e/features/home.feature

```diff
@@ -11,7 +11,7 @@ Feature: Home
     Given a non logged user
     When navigates to "/"
     And clicks on "Schedule a fishbowl" link
-    Then gets redirect to "/register"
+    Then gets redirect to "/fishbowl/create"
```

Comment:
That shouldn't be correct. When you are not logged, you don't create a fishbowl directly, first you go to register page.

Reply:
I think right now it goes to create, then it redirects to register, so maybe we should put both?

Reply:
The truth is that there are no redirects here since it is a webapp. What it is for sure is that it does not go to /fishbowl/create, or atleast that not the objective of the test (which is to ensure we end up in /register).